packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -3561,9 +3585,30 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin, Res
       SystemNavigator.selectSingleEntryHistory();
     }
 
+    SystemChannels.accessibility.setMessageHandler((dynamic message) => handleAccessibilityMessage(message as Object));
```

Comment:
just want to make sure i understand it right: 

1. setMessageHandler in `ServicesBinding`
2.  add something like `didReceiveAccessibilityMessage` to WidgetsBindingObserver
3.  call observer.didReceiveAccessibilityMessage() in `WidgetsBinding`
5. implement  WidgetsBindingObserver in Navigator

Reply:
Sorry I meant just general addListener/removelistener mechanism. WidgetBinderObserver may be too public facing for this api. I think you can create a similar structure as RelayoutWhenSystemFontsChangeMixin

Reply:
I still didn't get it.  RelayoutWhenSystemFontsChangeMixin is a mixin on RenderObject. By `create a similar structure as RelayoutWhenSystemFontsChangeMixin` do you mean creating a new mixin on ServicesBinding or a new mixin on Navigator?

Since ServicesBinding is already a mixin on BindingBase.  The closest thing I can think of is 
1. In Navigator, call  `ServicesBinding.instance.setAccessibilityMessageHandler()`
2.In ServicesBinding, call  `SystemChannels.accessibility.setMessageHandler()` in `setAccessibilityMessageHandler()`